libraries/AP_RangeFinder/AP_RangeFinder_Backend.h

```diff
@@ -69,6 +69,8 @@ class AP_RangeFinder_Backend
     // quality is not available
     virtual bool get_signal_quality_pct(uint8_t &quality_pct) const { return false; }
 
+    RangeFinder::Type allocated_type() const { return _backend_type; }
```

Comment:
maybe put a comment above this method just to save the ready 2 sec in figuring it out?